### ISSUE TITLE
Raise the cache max-age of static assets

### ISSUE THREAD

**Issue by edmorley** [taskcluster/unified-logviewer#27]
The static assets are currently being served with a low max-age, when they could be served with something much higher:

```
$ curl -Is 'https://taskcluster.github.io/unified-logviewer/0c3e17772699f2385f94.worker.js'  | grep 'Cache-Control'
Cache-Control: max-age=600
```

Unfortunately GitHub pages doesn't provide support for this, so hosting would need to be performed elsewhere (eg Heroku).

As part of doing this, more non-html assets would need to have cache-busting filenames (ie including a hash or version string), for example:
https://taskcluster.github.io/unified-logviewer/commons.js

**Comment by eliperelman** [comment]
I agree, and am honestly not sure why the commons chunk isn't using a hash-based filename as well.

@helfi92 this will probably need to be done inside a neutrino preset for the optimization work.

**Comment by eliperelman** [comment]
For reference:

https://github.com/mozilla-neutrino/neutrino-preset-web/blob/master/src/index.js#L43

**Comment by helfi92** [comment]
Thanks @eliperelman for the reference. I started the migration to Webpack 2 + optimization. Work in progress :)

**Comment by eliperelman** [comment]
Q2 2017 I plan on making this into a proper React component in a new repo that can be imported in TaskCluster and Treeherder, so this will be at the whim of the importer.
